client/src/components/registration/RegistrationForm.js

```diff
@@ -47,30 +56,32 @@ const RegistrationForm = () => {
     }
 
     setErrors(newErrors);
+    return _.isEmpty();
```

Comment:
there's a better way we can handle this - let me know when you're resolving and I can lend a hand!